test/clj_sqs_extended/core_test.clj

```diff
@@ -58,10 +69,12 @@
   (testing "Receive a message again correctly after (fake) processing problems"
     (let [client (fixtures/localstack-sqs)
           url (fixtures/test-queue-url)]
-      (sqs-ext/send-message client url test-message-1)
-      (let [message (sqs-ext/receive-message client url {:visibility-timeout 1})]
-        (is (= test-message-1 (:body message))))
-      (Thread/sleep 2000)
-      (let [message (sqs-ext/receive-message client url)]
-        (is (= test-message-1 (:body message)))))))
-
+      (doseq [format [:transit :json]]
```

Comment:
it's nice to have this for coverage. i'm on the fence about this though as we're testing functionality of SQS which perhaps we can assume they'd work as advertised. for sake of minimizing codebase.

Reply:
I'm really sorry, but I don't get what you'd like to minimize exactly.

Reply:
on second thought, nevermind @daisybytes